requirements/base.txt

```diff
@@ -1,2 +1,9 @@
-# Base requirements are empty. All of our minimum requirements are specified in setup.py's install_requires variable!
--e .
+django-filter==0.13.0
+django-js-reverse==0.7.2
+six==1.10.0
+djangorestframework==3.3.3
+django==1.9.7
+colorlog==2.6.0
+docopt==0.6.2
+django-mptt==0.8.5
+https://github.com/cherrypy/cherrypy/archive/v6.2.0.zip
```

Comment:
Looks like the restriction is still there (see Py Version column): 

![screen shot 2016-07-25 at 13 50 19](https://cloud.githubusercontent.com/assets/191955/17116936/dac262a2-526e-11e6-8953-34117c855d26.png)